.evergreen/README.md

```diff
@@ -72,6 +72,26 @@ specified in the `docker run` invocation and are not part of the image
 created by `docker build`. Because of this, they override any environment
 variables provided as positional arguments.
 
+### Field-Level Encryption FLE
+
+The Docker testing script supports running tests with field-level encryption (FLE).
+To enable FLE, set the FLE environment variable to true.
+
+Some FLE tests require other environment variables to be set as well. You may
+specify these environment variables in a private .env file as explained in the
+[Private Environment Variables](#private-environment-variables) section.
+
+The following is a list of required environment variables:
+- MONGO_RUBY_DRIVER_AWS_KEY
+- MONGO_RUBY_DRIVER_AWS_SECRET
+- MONGO_RUBY_DRIVER_AWS_REGION
+- MONGO_RUBY_DRIVER_AWS_ARN
+- MONGO_RUBY_DRIVER_MONGOCRYPTD_PORT
```

Comment:
Is/can the port be defaulted such that most people wouldn't need to set it?

Reply:
Oh whoops this port already has a default. I'll remove this.